cmd/ui/src/App.tsx

```diff
@@ -14,47 +14,92 @@
 import { ROUTE_EXPIRED_PASSWORD, ROUTE_LOGIN, ROUTE_USER_DISABLED } from 'src/ducks/global/routes';
 import { featureFlagKeys, getFeatureFlags } from 'src/hooks/useFeatureFlags';
 import { useAppDispatch, useAppSelector } from 'src/store';
 import { initializeBHEClient } from 'src/utils';
 import Content from 'src/views/Content';
-import Header from 'src/components/Header';
-
-const useStyles = makeStyles((theme) => ({
-    applicationContainer: {
-        display: 'flex',
-        position: 'relative',
-        flexDirection: 'column',
-        height: '100%',
-        overflow: 'hidden',
-    },
-    applicationHeader: {
-        flexGrow: 0,
-        zIndex: theme.zIndex.drawer + 1,
-    },
-    applicationContent: {
-        flexGrow: 1,
-        overflowY: 'auto',
-        overflowX: 'hidden',
-    },
-}));
+import Notifier from './components/Notifier';
 
-const App: React.FC = () => {
-    const classes = useStyles();
+const Inner: React.FC = () => {
+    const dispatch = useAppDispatch();
     const authState = useAppSelector((state) => state.auth);
     const queryClient = useQueryClient();
-    const dispatch = useAppDispatch();
     const location = useLocation();
 
+    const useStyles = makeStyles((theme) => ({
+        applicationContainer: {
+            display: 'flex',
+            position: 'relative',
+            flexDirection: 'column',
+            height: '100%',
+            overflow: 'hidden',
+            '@global': {
+                '.api-explorer .swagger-ui': {
+                    [`& a.nostyle,
+                        & div.renderedMarkdown > p,
+                        & .response-col_status,
+                        & .col_header,
+                        & div.parameter__name,
+                        & .parameter__in,
+                        & div.opblock-summary-description,
+                        & div > small,
+                        & li.tabitem,
+                        & .response-col_links,
+                        & .opblock-description-wrapper > p,
+                        & .btn-group > button,
+                        `]: {
+                        color: theme.palette.color.primary,
+                    },
+                    '& .responses-inner': {
+                        [`& h4, & h5`]: {
+                            color: theme.palette.color.primary,
+                        },
+                    },
+                    '& svg.arrow': {
+                        fill: theme.palette.color.primary,
+                    },
+                    '& .opblock-deprecated': {
+                        '& .opblock-title_normal': {
+                            color: theme.palette.color.primary,
+                        },
+                    },
+                },
+            },
+        },
+        applicationHeader: {
+            flexGrow: 0,
+            zIndex: theme.zIndex.drawer + 1,
+        },
+        applicationContent: {
+            backgroundColor: theme.palette.neutral.primary,
```

Comment:
I agree and feel like we could use one of the colors above the secondary defined neutral color so that there is still enough contrast between the background and the elements directly over the background. This was based on the given design but if there is a different consensus then I am happy to update this.

Reply:
Makes sense!